TU20Bot/Handler.cs

```diff
@@ -137,7 +137,9 @@ public class Handler {
                         $"Hey <@{message.Author.Id}>.\n\n" +
                         "To finish submitting your event, " +
                         "please react to your original message with one of the following tags. " +
-                        "React with ✅ to this prompt to dismiss it.\n\n" +
+                        "React with ✅ to this prompt to confirm your choices."+
+                        "React with ❌ to this prompt to cancel this action.\n\n" +
+                        "**Currently Available Tags (more will be added later):**\n" +
```

Comment:
The user needs to know that more tags will be added later? Saying "Currently Available" is enough in my opinion.

Reply:
P.S. Remove the double spacing in between tags for me please. I looked at it this morning and I don't like it.

Reply:
> Adding more information is helpful and friendly for our users as they are included in the progression of the tool over time.
Not all information is useful or helpful. I would argue what would the user actually use this information for? The prompt isn't going to change or update when new tags are added.

> It would be great if there was a spacing option between 1 and 2.
An inline field sounds perfect. It will look much better and its simpler than string concat. How about calling WithFields on embed builder?

Reply:
That's right! Adding information is a great way to make the tone of the content to be inclusive and friendly while providing additional information that someone could find helpful. For instance, in the latest update I changed the message to include a call-to-action, which is helpful for everyone to know that we are looking for their input.

Its a suggestion to use inline fields!

Reply:
```
                        "Do you have an idea for more tags? *Let us know!*"+
```

Okay, this is better... I think I would need to know how I would let the bot developers know? Maybe put your discord information here or something.